frontend/app/services/EmailService.scala

```diff
@@ -14,18 +19,27 @@ trait EmailService extends LazyLogging {
   val feedbackAddress: String
 
   val client = {
-    val c= new AmazonSimpleEmailServiceClient()
+    val c = new AmazonSimpleEmailServiceClient()
     c.setRegion(Region.getRegion(Regions.EU_WEST_1))
     c
   }
```

Comment:
You can write this as `Hashing.md5().hashString("your input", StandardCharsets.UTF_8).toString` <- note the `toString` method returns a nice hex string...

...or alternatively you could construct the blacklisted email as a `HashCode` object using `HashCode.fromString()`, and pass that around as a more strongly typed alternative to strings..!